package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "browserstack-local",
-  "version": "1.4.5",
+  "version": "1.4.6",
```

Comment:
Why is it here?

Reply:
Version bump for publishing in new package.